code/modules/projectiles/guns/projectile/automatic/lmg.dm

```diff
@@ -25,6 +25,7 @@
 	one_hand_penalty = 30 //you're not Stallone. LMG level.
 	spawn_blacklisted = TRUE
 	rarity_value = 80
+	gun_parts = list(/obj/item/part/gun = 1 ,/obj/item/stack/material/steel = 4)
```

Comment:
funnily enough steel isn't even used in building LMGs and dismantling them shouldn't yield any